.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+
+💐
+
+/cc @zendesk/dingo
+
+### Description
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
+
+#### Before merging this PR
+- [ ] Fill out the Risks section
+- [ ] Get two approvals from the Apps teams
```

Comment:
`Apps` -> `Extensibility`

Reply:
This is still an apps-specific gem, right? I wanted to try and keep this specific - API Experience are an Extensibility team, but they aren't the owners of this gem.